client/payment-details/summary/index.tsx

```diff
@@ -210,6 +210,8 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 	// present, partial refund is not possible.
 	const isPartiallyRefundable = charge.order && charge.order.number;
```

Comment:
Should we have a condition for this that checks whether the amount refunded is less than captured amount or is it already handled? 🤔

Reply:
Thanks, Deepak! Great point. I believe we're covering that check indirectly with the `showControlMenu` flag, which decides if the `DropdownMenu` shows up. This part of the code: https://github.com/Automattic/woocommerce-payments/blob/4d3de90979b5017a7cc3a4f41a0b74bc584ed1cf/client/payment-details/summary/index.tsx#L215, takes care of it by controlling when the charge is refunded.

So, we've got it covered, just in a roundabout way. Does that make sense, or do you think we should add a direct check for clarity?

Reply:
Thanks for sharing. I think that makes sense for now so I'll leave it as is, once we decide to add more items to the control menu, we can update then.